src/Symfony/Component/HttpFoundation/JsonResponse.php

```diff
@@ -59,6 +59,14 @@ public static function create($data = null, $status = 200, $headers = array())
     }
 
     /**
+     * Make easier the creation of JsonResponse from raw json
+     */
+    public static function fromJsonString($data = null, $status = 200, $headers = array())
```

Comment:
Btw `createFromString` somewhat implies a callback (JSONP) is parsed as well.. so you know ;-)